eras/shelley/impl/src/Cardano/Ledger/Shelley/RewardUpdate.hs

```diff
@@ -72,9 +72,8 @@ import NoThunks.Class (NoThunks (..), allNoThunks)
 -- ===============================================================
 
 -- | The result of reward calculation is a pair of aggregate Maps.
-data RewardAns c
-  = RewardAns
-      !(Map (Credential 'Staking c) (Reward c))
+newtype RewardAns c
+  = RewardAns (Map (Credential 'Staking c) (Reward c))
```

Comment:
why did you remove the strictness annotation?

Reply:
Because I changed it to a `newtype` :)

Reply:
ah! :)